static/js/top_left_corner.js

```diff
@@ -0,0 +1,97 @@
+var top_left_corner = (function () {
+
+var exports = {};
+
+exports.get_global_filter_li = function (filter_name) {
+    var selector = "#global_filters li[data-name='" + filter_name + "']";
+    return $(selector);
+};
+
+exports.update_count_in_dom = function (unread_count_elem, count) {
+    var count_span = unread_count_elem.find('.count');
+    var value_span = count_span.find('.value');
+
+    if (count === 0) {
+        count_span.hide();
+        value_span.text('');
+        return;
+    }
+
+    count_span.show();
+    value_span.text(count);
+};
```

Comment:
I think this doesn't need to be exported; the `stream_list` one was apparently only exported for one test..